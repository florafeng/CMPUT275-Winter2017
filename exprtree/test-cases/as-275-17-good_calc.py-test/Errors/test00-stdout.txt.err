Expected file "test-cases/as-275-16-good_calc.py-test/Expected/test00-stdout.txt.err" has no output file match
