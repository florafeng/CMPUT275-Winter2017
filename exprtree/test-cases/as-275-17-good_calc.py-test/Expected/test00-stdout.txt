Unparse: 42
Result: 42
Unparse: 1
Result: 1
Unparse: (1 + 2)
Result: 3
Done
